src/frontend/src/views/PrivacyPreview/index.tsx

```diff
@@ -0,0 +1,1217 @@
+import { List, ListItem, ListItemLabel } from "czifui";
+import { HeadAppTitle } from "src/common/components";
+import { NewTabLink } from "src/common/components/library/NewTabLink";
+import { ROUTES } from "src/common/routes";
+import {
+  B,
+  H1,
+  H2,
+  H3,
+  H4,
+  NarrowContainer,
+  Number,
+  P,
+  Title,
+} from "src/common/styles/basicStyle";
+import { PageContent } from "../../common/styles/mixins/global";
+import {
+  ContentRow,
+  H2AnchorLink,
+  SectionRow,
+  Table,
+  TopRow,
+  UnderLineHeader,
+} from "./style";
+
+const PrivacyPolicyPreview = (): JSX.Element => {
+  const renderIntro = () => (
+    <>
+      <Title>
+        <H1>Chan Zuckerberg GEN EPI (formerly Aspen) Privacy Policy</H1>
+        <H4>Last Updated: January 1, 2023</H4>
```

Comment:
@Janeece  The announcements banner says "We are updating our Privacy Policy to comply with CCPA, effective January 3, 2023." but this date is set for Jan 1. Seems like they should match?

Also, it feels to me like the preview privacy policy page should indicate somewhere that it is currently only a **preview** and has not actually gone into force yet. Along those lines, maybe shouldn't say "Last Updated", but rather "will take effect on" or something? That's more of a legal question though.

Anyway, neither of those are code issues, but wanted to flag them still.

Reply:
Yeah, lets set the date to match when it will go live (Jan 1). I'm not so sure about the "last updated", I agree that its more of a legal question — I'd check in with Olivia/Christine about that since they were a part of those conversations with legal.